Alicerce/Sources/Logging/Log+ConsoleProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleProvider: LogProvider {
+
+        public enum ConsoleOutput {
+            case print
+            case nslog
+        }
+
+        public var minLevel = Log.Level.error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
+        public var output = ConsoleOutput.print
```

Comment:
@rbarbosa i agree to use type inference, this is a eye candy thing, for example, if a struct has no default values, you can't infer the type and if for some reason the inferred type is not what you want, you will have some properties with inference and others don't.

```
struct LogItem {
    let format: String
    let date: Date
}
```
i like this one ☝️ 

```
class LogItem {
    var format = Formatter.default
    var date = Date()
}
```
i also like this one ☝️ 

but,

```
class LogItem {
    var format = Formatter.default
    var date: Date

    ...
}
```
but i don't like this one ☝️ 

that is my two cents.